argocd-image-updater/Dockerfile

```diff
@@ -10,15 +10,15 @@ WORKDIR /work/argocd-image-updater
 RUN mkdir -p dist && \
     make controller
 
-FROM quay.io/cybozu/ubuntu:22.04
+FROM ghcr.io/cybozu/ubuntu:22.04
 LABEL org.opencontainers.image.source="https://github.com/cybozu/neco-containers"
 
 RUN groupadd -g 10000 argocd && \
     useradd -r -u 10000 -g argocd argocd && \
     mkdir -p /home/argocd && \
     chown argocd:argocd /home/argocd && \
     apt-get update && \
-    apt-get install -y --no-install-recommends git openssh-client python3 python3-pip && \
+    apt-get install -y --no-install-recommends git openssh-client python3 python3-pip tini gpg gpg-agent && \
     pip3 install --upgrade pip && \
     pip3 install awscli && \
```

Comment:
`24.04/ubuntu-debug` installs awscli with `curl`, how about using this method?
With this, we can omit installing `python3` and `python3-pip` I think.
https://github.com/cybozu/ubuntu-base/blob/1cb03b74e6b001c989ea83296a035e1bb9b7d129/24.04/ubuntu-debug/Dockerfile#L55-L58